tsl/test/expected/move.out

```diff
@@ -255,7 +255,6 @@ RESET client_min_messages;
 -- move with chunk index for reorder
 SELECT true INTO move_chunk_result FROM move_chunk(chunk=>'_timescaledb_internal._hyper_1_2_chunk', destination_tablespace=>'tablespace1', index_destination_tablespace=>'tablespace1', reorder_index=>'_timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx', verbose=>TRUE);
 INFO:  reordering "_timescaledb_internal._hyper_1_2_chunk" using sequential scan and sort
-INFO:  "_hyper_1_2_chunk": found 0 removable, 5 nonremovable row versions in 1 pages
```

Comment:
Since this is a move_chunk, it does not seem like the `INFO` output needs to be part of the test. Either the chunk was moved successfully, or it was not, but if reordering was done and if the table was rebuilt does not seem to be relevant, so you can probably change this to `WARNING` level.

Reply:
I think the reason to verbose the output here is to do a reordering of the chunk based on the given index, but is strange that the info output is `using sequential scan and sort`.